Childrens-Social-Care-CPD/Views/Shared/_PageBanner.cshtml

```diff
@@ -4,17 +4,31 @@
 
 @if (Model != null && Model.ShowContentHeader)
 {
+    ContextModel contextModel = (ContextModel)ViewBag.ContextModel;
+
     <section id="content-banner" class="app-section-content app-section-content--blue govuk-!-margin-bottom-8">
         <div class="dfe-width-container">
             <div class="govuk-grid-row govuk-!-padding-top-3 govuk-!-padding-bottom-0">
                 <div class="govuk-grid-column-two-thirds-from-desktop banner-breadcrumbs">
-                    <partial name="_BreadcrumbTrail" />
+                    <partial name="_BreadcrumbTrail" model="Model" />
                 </div>
                 <div class="govuk-grid-column-two-thirds-from-desktop govuk-!-padding-top-3">
                     <h1 id="content-banner-title" class="govuk-heading-xl">@Model.ContentTitle</h1>
                     <p id="content-banner-subtitle" class="govuk-body-l">@Model.ContentSubtitle</p>
                 </div>
+                @if (Model.PageType == PageType.PathwaysTrainingContent)
```

Comment:
Query - should this location section be separated out into a partial of it's own? Or are we happy with the code living in the Page Banner partial?  The breadcrumb trail is in its own pathway ...

Reply:
Have separated this out into separate partial